admin/src/templates/targets.html

```diff
@@ -36,7 +36,8 @@
       <div class="row selection" ng-repeat="target in configuration.items track by target.id" ng-click="edit(target.id)">
         <div class="col-xs-1" ng-bind-html="target.icon | resourceIcon"></div>
         <div class="col-xs-4">{{target.id}}</div>
-        <div class="col-xs-4">{{target.name | translateFrom}}</div>
+        <div class="col-xs-4" ng-if="target.name">{{target.name | translateFrom}}</div>
+        <div class="col-xs-4" ng-if="target.translation_key">{{target.translation_key | translate}}</div>
```

Comment:
I just realised there's one more issue with this... it's possible to end up with both of these columns which breaks the bootstrap row. I think it should be like this...

```js
<div class="col-xs-4" ng-if="target.translation_key">{{target.translation_key | translate}}</div>
<div class="col-xs-4" ng-if="!target.translation_key">{{target.name | translateFrom}}</div>
```

That way you always get one and only one of the divs.

Reply:
Yeah I thought so but I was debating with myself what are chances of this occurring? but anyway we cannot make assumptions